packages/integration_test/lib/fix_data/README.md

```diff
@@ -0,0 +1,45 @@
+## Directory contents
+
+The `.yaml` files in these directories are used to
+define the [`dart fix` framework](https://dart.dev/tools/dart-fix) refactorings
+used by `integration_test`.
+
+The number of fix rules defined in a file should not exceed 50 for better
+maintainability. Searching for `title:` in a given `.yaml` file will account
+for the number of fixes. Splitting out fix rules should be done by class.
+
+When adding a new `.yaml` file, make a copy of `template.yaml`. Each file should
+be for a single class and named `fix_<class>.yaml`. To make sure each file is
+grouped with related classes, a `fix_<filename>` folder will contain all of the
+fix files for the individual classes.
+
+See the flutter/packages/integration_test/test_fixes directory for the tests
+that validate these fix rules.
+
+To run these tests locally, execute this command in the
+flutter/packages/integration_test/test_fixes directory.
+```sh
+dart fix --compare-to-golden
+```
+
+For more documentation about Data Driven Fixes, see
+https://dart.dev/go/data-driven-fixes#test-folder.
+
+To learn more about how fixes are authored in package:integration_test, see
+https://github.com/flutter/flutter/wiki/Data-driven-Fixes
+
+## When making structural changes to this directory
+
+The tests in this directory are also invoked from external
+repositories. Specifically, the CI system for the dart-lang/sdk repo
+runs these tests in order to ensure that changes to the dart fix file
+format do not break Flutter.
```

Comment:
Is this true for the integration test ones as well?

Reply:
Not yet, but I did write up [dart #52795](https://github.com/dart-lang/sdk/issues/52795) to start that discussion with the dart team. We had the same question and resolution when we added dart fix to `flutter_test` as well.